doc/development_process/gh_labels.rst

```diff
@@ -0,0 +1,128 @@
+.. _gh_labels:
+
+Labeling issues and pull requests in GitHub
+###########################################
+
+Lots of GitHub issues and pull requests are being created daily.
+To help us manage them we have “labels”:
+to classify them by area, type, priority, or to mark them in need of attention.
+
+All GitHub issues or pull requests must be appropriately labeled.
+When reviewing a PR, if it has missing or incorrect labels, maintainers shall
+fix it.
+
+This saves us all time when searching, reduces the chances of the PR or issue
+being forgotten, speeds up reviewing, reduces the changes of duplicates, etc.
+
+These are the labels we currently have, grouped by type:
+
+Area:
+*****
+
+=============  ===============================================================
+Labels         ``area:*``
+Applicable to  PRs  and issues
+Description    Indicates which areas are affected by the bug or pull request
+=============  ===============================================================
+
+An area maintainer should be able to filter by his area and
+find all issues and PRs which relate to that area.
+
+Platform
+*********
+
+=============  ===============================================================
+Labels         ``platform:*``
+Applicable to  PRs  and issues
+Description    An issue or PR which affects only a particular platform
+=============  ===============================================================
+
+To be discussed in a meeting
+****************************
+
+=============  ===============================================================
+Labels         ``API``, ``dev-review``, ``TSC``
+Applicable to  PRs  and issues
+Description    The issue is to be discussed in the following API/dev-review/TSC
+               meeting if time permits.
+=============  ===============================================================
+
+Minimum PR review time
+**********************
+
+=============  ===============================================================
+Labels         ``Hot Fix``, ``Trivial PR``, ``Maintainer``,
+               ``Security Review``, ``TSC``
+Applicable to  PRs only
+Description    Depending on the PR complexity, an indication of how long a merge
+               should be held to ensure proper review. See
+               :ref:`review process <review_time>`
+=============  ===============================================================
+
+Issue priority labels
+*********************
+
+=============  ===============================================================
+Labels         ``priority:{high|medium|low}``
+Applicable to  Issues only
+Description    To classify the priority/importance of a bug or feature
+=============  ===============================================================
+
+Note: Issue priorities are decided/changed during the bug-triage or TSC
```

Comment:
```suggestion
Note: Issue priorities are generally set or changed during the issue triage or TSC.
```

Reply:
I take the 1st change. About the 2nd, the meeting invite actually calls it "Bug Triage & Release Readiness", so I was trying to be consistent with that.